vcu118-run.py

```diff
@@ -286,6 +301,11 @@ def load_and_start_kernel(*, gdb_cmd: Path, openocd_cmd: Path, bios_image: Path,
     args += ["-ex", "set style enabled off"]  # disable colours since they break the matcher strings
     args += ["-ex", "monitor reset init"]  # reset and go back to boot room
     args += ["-ex", "si 5"]  # we need to run the first few instructions to get a valid DTB
+    for core in range(1, n_cores):
+        args += ["-ex", "thread {:d}".format(core + 1)] # switch to thread (core + 1) (GDB counts from 1)
+        args += ["-ex", "si 5"] # execute bootrom on every core
+        args += ["-ex", "set $pc=0xc0000000"] # set every core to start of cached memory XXX Assumes this the start of bios
```

Comment:
If you want to do this properly, you could do:
```
set $entry_point=$pc
thread 2
set $pc=$entry_point
thread 3
set $pc=$entry_point
...
```
(as `-ex` commands) after the `load` below instead, I believe.